Index: .idea/NTQQBots.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"PYTHON_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\" />\r\n    <orderEntry type=\"jdk\" jdkName=\"Python 3.11 (NTQQBots)\" jdkType=\"Python SDK\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/NTQQBots.iml b/.idea/NTQQBots.iml
--- a/.idea/NTQQBots.iml	(revision 27116df04eecdcaac54fde8ed9311a30682676dc)
+++ b/.idea/NTQQBots.iml	(date 1710082850635)
@@ -5,4 +5,8 @@
     <orderEntry type="jdk" jdkName="Python 3.11 (NTQQBots)" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
+  <component name="PackageRequirementsSettings">
+    <option name="versionSpecifier" value="不指定版本" />
+    <option name="keepMatchingSpecifier" value="false" />
+  </component>
 </module>
\ No newline at end of file
